flake.nix

```diff
@@ -18,20 +18,22 @@
   inputs.fenix.url = "github:nix-community/fenix";
```

Comment:
@alxiong I guess you still have another formatter running?

Reply:
do you mean another nix formatter? I don't think so, totally switch to `nil` since our last chat about it.
are you suggesting we remove this line?

Reply:
My comment just refers to the fact that the file has been reformatted, so I assumed it was formatted by a different formatter than `nixpkgs-fmt`.